src/views/Publish.vue

```diff
@@ -150,6 +159,55 @@ export default defineComponent({
       apiVersion: RequestsUtils.confAPIVersion,
       // loading indicator
       isPublishLoading: false,
+
+      // Publish History
+      columns: [
+        {
+          title: 'Full Date',
+          fieldNames: ['full_date'],
+          isSortByOriginalValue: true,
+          displayFunction: (item: any) => {
+            const newDate = new Date(item['full_date'])
+            return DateTimeUtils.isoToNowFullCuriefenseFormat(newDate)
+          },
+          isSortable: true,
+          isSearchable: true,
+          classes: 'ellipsis width-170px',
+        },
+        {
+          title: 'Source Branch',
+          fieldNames: ['source_branch'],
+          isSortable: true,
+          isSearchable: true,
+          classes: 'width-130px',
+          cellContentClasses: 'ellipsis',
+        },
+        {
+          title: 'Target Bucket',
+          fieldNames: ['target_bucket'],
+          displayFunction: (item: PublishHistory) => {
+            return item.target_bucket?.join('\n')
+          },
+          isSortable: true,
+          isSearchable: true,
+          classes: 'width-130px',
+        },
+        {
+          title: 'Commit hash ID',
+          fieldNames: ['commit_hash_id'],
+          isSortable: true,
+          isSearchable: true,
+          classes: 'ellipsis',
+        },
+        {
+          title: 'Published User',
```

Comment:
`Author` would fit better here as it is the correct git term
```suggestion
          title: 'Author',
```